packages/Search/src/details/DTK_DetailsUtils.hpp

```diff
@@ -54,6 +54,22 @@ void exclusive_prefix_sum( Kokkos::View<T *, DeviceType> in,
     Kokkos::fence();
 }
 
+/** \brief Get last element.
+ *
+ *  Returns a copy of the last element in the view.
+ *
+ *  \pre \c in is not empty.
+ */
+template <typename T, typename DeviceType>
+T last_element( Kokkos::View<T *, DeviceType> in )
```

Comment:
Either make the function work on the device or make it clear in the name of the function that it is supposed to be used on the host.